app/domain/authentication/authn_k8s/k8s_host.rb

```diff
@@ -12,22 +13,30 @@
 
 module Authentication
   module AuthnK8s
+
+    Err ||= Errors::Authentication::AuthnK8s
+
     class K8sHost
       extend Forwardable
 
       attr_reader :account, :service_name, :csr
 
       def self.from_csr(account:, service_name:, csr:)
         cn = csr.common_name
-        raise ArgumentError, 'CSR must have a CN entry' unless cn
-
+        Rails.logger.debug(Err::CSRMissingCNEntry.new(csr.subject_to_s, csr.spiffe_id.to_s))
```

Comment:
This is redundant. The message will be written to the log once it is rescued in `authenticate_controller.rb`

Reply:
my bad